server/models/dtos/project_dto.py

```diff
@@ -197,12 +195,12 @@ class ProjectDTO(Model):
     )
     mapping_permission = StringType(
         required=True,
-        serialized_name="mapping_permission",
+        serialized_name="mappingPermission",
```

Comment:
I would suggest to make that change... after your fixes, I'll commit a change in the frontend